jsk_naoqi_robot/README.md

```diff
@@ -37,12 +51,6 @@ rosdep install -y -r --from-paths src --ignore-src
 catkin build
```

Comment:
Before executing `catkin build`, installing `ros-$ROS_DISTRO-pepper(nao)-meshes` manually is required in order to get `pepper(nao).l`